modules/extensions-spring-support/src/test/resources/global-pojo.xsd

```diff
@@ -29,49 +29,49 @@
             </xs:complexContent>
         </xs:complexType>
     </xs:element>
-    <xs:complexType name="org.mule.runtime.module.extension.internal.runtime.connector.basic.Owner">
+    <xs:complexType name="org.mule.runtime.module.extension.internal.runtime.connectivity.basic.Owner">
         <xs:complexContent>
             <xs:extension xmlns:mule="http://www.mulesoft.org/schema/mule/core" base="mule:abstractExtensionType">
-                <xs:attribute type="mule:expressionString" use="required" name="requiredFieldExpressionSupported"></xs:attribute>
-                <xs:attribute type="mule:expressionString" use="optional" name="optionalFieldExpressionRequired"></xs:attribute>
-                <xs:attribute type="mule:expressionString" use="optional" name="optionalFieldDefault"></xs:attribute>
                 <xs:attribute type="xs:string" use="required" name="requiredFieldExpressionNotSupported"></xs:attribute>
                 <xs:attribute type="mule:expressionString" use="required" name="requiredFieldDefault"></xs:attribute>
                 <xs:attribute type="xs:string" use="optional" name="optionalFieldExpressionNotSupported"></xs:attribute>
                 <xs:attribute type="mule:expressionString" use="required" name="requiredFieldExpressionRequired"></xs:attribute>
                 <xs:attribute type="mule:expressionString" use="optional" name="optionalFieldExpressionSupported"></xs:attribute>
+                <xs:attribute type="mule:expressionString" use="required" name="requiredFieldExpressionSupported"></xs:attribute>
```

Comment:
That has to be fixed, otherwise this files will be constantly changed. Can you fill a jira for that?